app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ResourcesController < ApplicationController
+  before_action :set_resource, only: [:show]
+
+  def show
+    # TODO: Switch this to suggestion mechanism once implemented
+    @suggestions = Group.limit(6)
```

Comment:
This may also propose different items, like other Resources, Lists, People. Is that what you're envisioning here? (No change needed for this PR, but just wanted to make sure that was clear)